ports/rp2/mphalport.c

```diff
@@ -84,21 +89,10 @@ uintptr_t mp_hal_stdio_poll(uintptr_t poll_flags) {
 // Receive single character
 int mp_hal_stdin_rx_chr(void) {
     for (;;) {
-        #if MICROPY_HW_ENABLE_UART_REPL
         int c = ringbuf_get(&stdin_ringbuf);
```

Comment:
will probably need to check cdc pending here and call `tud_cdc_rx_cb`, because the poll function may never be called

Reply:
@hoihu I still think this comment needs attention.  If `cdc_itf_pending` is set then this function may run forever and never check that flag.

Reply:
@dpgeorge I pushed an update. Since `mp_hal_stdio_poll()` and `mp_hal_stdin_rx_chr()` share the same logic, I moved the common code to a separate function `poll_cdc_interfaces()`.